modules/ROOT/pages/autocorrect.adoc

```diff
@@ -0,0 +1,44 @@
+= Spelling Autocorrect plugin
+:navtitle: Spelling Autocorrect
+:description: Autocorrect common spelling mistakes.
+:keywords: autocorrect, capitalization, spell, spelling, spellcheck
+:pluginname: Spelling Autocorrect
+:plugincode: autocorrect
+:pluginminimumplan: tiertwo
+
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+The {pluginname} plugin automatically corrects a specified set of spelling errors, capitalization errors, or both, as they are being typed.
+
+== Interactive Example
+
+liveDemo::autocorrect[]
+
+include::partial$misc/purchase-premium-plugins.adoc[]
+
+== Basic setup
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'autocorrect'
+});
+----
+
+== Options
+
+include::partial$configuration/autocorrect_autocorrect.adoc[leveloffset=+1]
+
+include::partial$configuration/autocorrect_capitalize.adoc[leveloffset=+1]
+
+include::partial$configuration/autocorrect_rpc_url.adoc[leveloffset=+1]
```

Comment:
This should not be documented so please remove this. It's an old option we aren't advertising/don't want customers to use.